linkerd2-proxy/src/rt.rs

```diff
@@ -0,0 +1,31 @@
+use tokio::runtime::{self, Runtime};
+
+#[cfg(feature = "multicore")]
+pub(crate) fn build() -> Runtime {
+    let builder = runtime::Builder::new()
+        .enable_all()
+        .thread_name("linkerd2-proxy-worker");
+    let num_cpus = num_cpus::get();
+    if num_cpus > 2 {
+        builder
+            .threaded_scheduler()
+            .core_threads(num_cpus - 1) // Save 1 core for the admin runtime.
```

Comment:
I don't think we should dedicate an entire core to the admin thread... That's *a lot* of resources for something we shouldn't be doing much work on. The real goal of the separate admin runtime is to ensure we can still interact with the proxy if, for instance, the main runtime isn't healthy. I think we're best off keeping it simple and treating it as if the admin thread needs no dedicated resources.

Reply:
Mmm, yeah, that makes sense. My thought process was that if forwarding has more workers, load on the admin runtime for e.g. discovery & DNS lookups might increase. I think the long term solution might be to just move that work off of the admin runtime, and have it _only_ run metrics/tap/trace endpoints/etc. Then we can probably feel confident not giving it a whole core.